util.go

```diff
@@ -11,6 +11,11 @@ import (
 	"github.com/mattn/go-runewidth"
 )
 
+type ExpandedLines struct {
```

Comment:
`ExpandedLines` is public but it must be private i.e. must be renamed to `expandLines`. Also a comment will be appreciated to know what it does.